clang/test/CheckedCRewriter/arrtestsBUG.c

```diff
@@ -0,0 +1,98 @@
+// XFAIL: *
```

Comment:
What is the reason for some of these tests failing? Do we intend to remove the XFAIL's in future?